c-for-go generating invalid go functions

I'll preface this by saying I'm very new to `c-for-go` and how it operates. I'm looking at updating some library bindings that currently work with an old release of the library, but when I point it at the updated headers for the library, I get errors like this:

```
  processing vulkan.yml ⠏[WARN] cannot gofmt ../vulkan/cgo_helpers.go: ../vulkan/cgo_helpers.go:42347:17: expected '(', found '.' (and 10 more errors)
```

looking at `cgo_helpers.go`, it looks like `c-for-go` has generated some invalid function declarations with periods in them, which neither `go build` or `gofmt` is very happy with:

```
func copyPUnsafe.PointerBytes(slice *sliceHeader) (*unsafe.Pointer, *cgoAllocMap)
func allocUnsafe.PointerMemory(n int) unsafe.Pointer
```

is there something extra I'm missing that I need to do so it generates these functions differently? what's going on here? :sweat_smile:

--- a/generator/gen_bindings.go
+++ b/generator/gen_bindings.go
@@ -75,6 +75,9 @@ func (gen *Generator) getTypedHelperName(goBase string, spec tl.CGoSpec) string 
 	if strings.HasPrefix(spec.Base, "C.") {
 		spec.Base = spec.Base[2:]
 	}
+	if strings.HasPrefix(spec.Base, "unsafe.") {
+		spec.Base = spec.Base[7:]
+	}
 	specName := gen.tr.TransformName(tl.TargetType, spec.Base)
 	buf := new(bytes.Buffer)
 	for _, size := range spec.OuterArr.Sizes() {

